tests/acceptance/features/apiSpaces/quota.feature

```diff
@@ -0,0 +1,75 @@
+@api @skipOnOcV10
+Feature: State of the quota
+  As a user
+  I want to be able to see the state of the quota and and not let the quota overrun:
+  quota state indication:
+  | 0 - 75%  | normal   |
+  | 76 - 90% | nearing  |
+  | 91 - 99% | critical |
+  | 100 %    | exceded  |
+
+  Note - this feature is run in CI with ACCOUNTS_HASH_DIFFICULTY set to the default for production
+  See https://github.com/owncloud/ocis/issues/1542 and https://github.com/owncloud/ocis/pull/839
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And the administrator gives "Alice" the role "Admin" using the settings api
+    
+
+  Scenario Outline: check the quota display 
+    When user "Alice" creates a space "<spaceName>" of type "project" with quota "<total>" using the GraphApi
+    And user "Alice" lists all available spaces via the GraphApi
+    And user "Alice" uploads a file inside space "<spaceName>" with content "<fileContent>" to "test.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
```

Comment:
```suggestion
    And user "Alice" has uploaded a file inside space "<spaceName>" with content "<fileContent>" to "test.txt" using the WebDAV API
```

It would be nice to have the `Given` form of step words here. And the `Given` step can have the check for `201` inside it. We usually write `Given` steps like this - they do some scenario setup, and do a reasonable check that the scenario setup action worked.